app/controllers/orga/seasons_controller.rb

```diff
@@ -48,16 +48,16 @@ def destroy
   # by opening and closing the corresponding links in the nav bar
   def switch_phase
     return if Rails.env.production?
-    @season = current_season
+    @switch = current_season.phase_switcher
```

Comment:
Ok, a bit of a crossroads here:

I would either

a) remove `Season#phase_switcher` from the the public interface, forward the `fake_*_phase` methods to a private switcher instance transparently and keep calling `fake_*_phase` on `current_season` 

or

b) leave `Season` completely out of the phase switching business by calling the class directly (see comment above). This would also mean moving the phase-switching examples into a separate spec file for `Season::PhaseSwitcher`.

I think b) is the better way to go, but I let you decide :)